tensorflow_addons/layers/adaptive_pooling.py

```diff
@@ -53,16 +53,13 @@ def __init__(
         super().__init__(**kwargs)
 
     def call(self, inputs, *args):
-        bins = self.output_size[0]
```

Comment:
Here bins probably has more semantic info.

Reply:
I agree.
Thank You!